src/tracer.js

```diff
@@ -54,11 +55,13 @@ export default class Tracer {
      *        as process-level tags on the Tracer itself.
      * @param {Object} [options.metrics] - instance of the Metrics class from ./metrics/metrics.js.
      * @param {Object} [options.logger] - a logger matching NullLogger API from ./logger.js.
+     * @param {Boolean} [upsampling] - determines whether to resample child spans of unsampled parents.
      */
     constructor(serviceName: string,
             reporter: Reporter = new NoopReporter(),
             sampler: Sampler = new ConstSampler(false),
-            options: any = {}) {
+            options: any = {},
+            upsampling: any = {}) {
```

Comment:
ES6 has a strange implementation of optional parameters.  Its not like python were I can specify which optional parameter I mean with a keyword.  So if optional parameter 1 is missing  you can't specify optional parameter 2.  Basically it lets nullable arguments not be passed, however they are not true optional parameters.